### PR TITLE
FAT-3638 Karate Tests: Linked Authority updates

### PR DESCRIPTION
**Pull request by Shans-Kaluhin**
image

### CODE REVIEW

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    Given path '/source-storage/records', bibId
+    When method GET
+    Then status 200
+    And match response.parsedRecord.content.fields[*].100.subfields[*].a contains only 'Updated'
+
+    # retrieve instance record
+    Given path '/instance-storage/instances', instanceId
+    When method GET
+    Then status 200
+    And match response.parsedRecord.content.fields[*].100.subfields[*].a contains only 'Updated'
```
Comment (psmagin) [inline mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature:47]: The instance doesn't have subfields

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+    # retrieve bib srs record - should delete authority link
+    Given path '/source-storage/records', bibId
+    When method GET
+    Then status 200
+    And match response.parsedRecord.content.fields[*].100 != []
+    And match response.parsedRecord.content.fields[*].100.authorityId == []
+    And match response.parsedRecord.content.fields[*].100.authorityNaturalId == []
+    And match response.parsedRecord.content.fields[*].100.subfields[*].0 == []
+    And match response.parsedRecord.content.fields[*].100.subfields[*].9 == []
```
Comment (psmagin) [inline mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature:104]: Also check that instance has authorityId=null

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature [hunk 3]
```diff
@@ -0,0 +1,104 @@
+Feature: linking-rules tests
```
Comment (psmagin) [inline mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature:1]: It's not linking-rules tests

**Review by psmagin — CHANGES_REQUESTED** [review]

File: mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature (hunk 1 above)
Comment (psmagin) [reply mod-quick-marc/src/main/resources/spitfire/mod-quick-marc/features/quick-marc-linking-records.feature:47]: How does it's even passed? Is this condition workable, do they really check smth?

**Review by psmagin — APPROVED** [review]

**Review by hasanepam — APPROVED** [review]

**Review by viacheslavkol — APPROVED** [review]

**Pull request merged**